web/popminer/api.go

```diff
@@ -206,3 +214,65 @@ type BitcoinUTXO struct {
 	// Value is the value of the output in satoshis.
 	Value int64 `json:"value"`
 }
+
+// EventType represents a type of event.
+type EventType string
+
+const (
+	// EventTypeMinerStart is dispatched when the PoP miner has started.
+	EventTypeMinerStart EventType = "minerStart"
+
+	// EventTypeMinerStop is dispatched when the PoP miner has exited.
+	EventTypeMinerStop EventType = "minerStop"
+
+	// EventTypeMineKeystone is dispatched when the PoP miner is mining an L2
+	// keystone.
+	EventTypeMineKeystone EventType = "mineKeystone"
+
+	// EventTypeTransactionBroadcast is dispatched when the PoP miner has
+	// broadcast a Bitcoin transaction to the network.
+	EventTypeTransactionBroadcast EventType = "transactionBroadcast"
+)
+
+// popmEvents contains events dispatched by the native PoP Miner.
+// These events will be forwarded to JavaScript, however we also dispatch events
+// that are specific to the WebAssembly PoP Miner.
+var popmEvents = map[popm.EventType]EventType{
+	popm.EventTypeMineKeystone:         EventTypeMineKeystone,
+	popm.EventTypeTransactionBroadcast: EventTypeTransactionBroadcast,
+}
+
+// eventTypes is a map used to parse string event types.
+var eventTypes = map[string]EventType{
+	"":                                     "", // Listen for all events.
```

Comment:
maybe use * ?

Reply:
I think that would be better, will update.